src/translations/en.json

```diff
@@ -70,6 +70,9 @@
     "emailaddress": "E-mail",
     "delete": "Delete",
     "delete-in-progress": "Being deleted",
+    "delete-in-progress-explanation": "Your e-mail address is marked for deletion and will be deleted within a few days.",
+    "revalidate-in-progress": "Being revalidated",
+    "revalidate-in-progress-explanation": "Your e-mail address is marked invalid. Does it still exist? Or is there a problem with your mail server? We will check again within 5 days. If it works, your e-mail address will be active again. If not, we will delete it for you.",
```

Comment:
I don't think the question `Or is there a problem with your mail server?` means much to users as very few of them will have a mailserver.

Can we somehow include the error that caused the `Being revalidated` ?

Reply:
That could indeed be a little too technical, good find! I've removed the 'mail server' part here and in the other translation parts and think it will be easier to read now.

About your question including the cause:
Since the error happened earlier in the automated task processing in the kss it would require to add a new column in the database where we need to store the error, then change the api interface so it returns the reason to the irma_keyshare_webclient. Then we would also have to think about possibly translating the possible errors. This all means many changes. Therefore it is best to use the more generic approach (for now) I think.